modules/custom/az_alphabetical_listing/az_alphabetical_listing.module

```diff
@@ -41,3 +85,90 @@ function az_alphabetical_listing_theme_registry_alter(&$theme_registry) {
     }
   }
 }
+
+/**
+ * Implements hook_views_post_execute().
+ */
+function az_alphabetical_listing_views_post_execute(ViewExecutable $view) {
+
+  // Check if we're on the correct view and display.
+  if (($view->id() === 'az_alphabetical_glossary') && ($view->current_display === 'az_alphabetical_glossary_attachment')) {
+    $entity_type_id = $view->getBaseEntityType()->id();
+    // Get the results of the current query.
+    /** @var \Drupal\views\ResultRow[] $results */
+    $results = $view->result ?? [];
+
+    // Get field_az_alphabetical_index storage settings for allowed values.
+    $allowed_values = [];
+
+    /** @var \Drupal\field\Entity\FieldStorageConfig $field_storage_config */
+    $field_storage_config = FieldStorageConfig::loadByName($entity_type_id, 'field_az_alphabetical_index');
+    if ($field_storage_config) {
+      $allowed_values = $field_storage_config->getSetting('allowed_values');
+    }
+
+    // Get the letters that were present in the original view results.
+    $letters_in_original_results = [];
+    // dpm($results);
+    $count = 0;
+    foreach ($results as $original_row_index => $original_row) {
+      if (isset($original_row->_entity)) {
+        /** @var \Drupal\Core\Entity\ContentEntityInterface $entity */
+        $entity = $original_row->_entity;
+        $field_items = $entity->get('field_az_alphabetical_index');
+        if (!$field_items->isEmpty()) {
+          // Use the actual field value (key) not the label.
+          $letters_in_original_results[$field_items->value] = $original_row->num_records;
+        }
```

Comment:
Since this field may or may not exist on a given content type, it seems necessary to check for its existence. A php error seems to result for this view if the field has not been added for the content type.
```suggestion
        if ($entity->hasField('field_az_alphabetical_index')) {
          $field_items = $entity->get('field_az_alphabetical_index');
          if (!$field_items->isEmpty()) {
            // Use the actual field value (key) not the label.
            $letters_in_original_results[$field_items->value] = $original_row->num_records;
          }
        }
```